app/models/topic_tracking_state.rb

```diff
@@ -242,7 +242,7 @@ def self.publish_read(topic_id, last_read_post_number, user_id, notification_lev
       }
     }
 
-    MessageBus.publish(self.unread_channel_key(user_id), message.as_json, user_ids: [user_id])
```

Comment:
This is a concern to me, we are moving everything to one channel here which is not ideal, all sorts of things can break, backlog can start behaving in non ideal ways. 

I am fine centralizing on `/topic-tracking-state/USER_ID` that way each user gets 1 dedicated channel with far more predictable backlogs and so on.

Reply:
This is a good point. I'll update the PR to use `/topic-tracking-state/USER_ID` instead.

Reply:
Ahh ok I see why we can't just split using the `user_id` because we don't always have a `user_id` to publish to on the server side. For example, `/latest` is published to both anon and logged in users. What about increasing the backlog for the channel instead? Technically, all the channels before centralisation has the potential to burst the backlog too if the site experiences high levels of user activity.